src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IForEachLoopStatement.cs

```diff
@@ -6031,7 +6031,7 @@ static void Main()
         Block[B4] - Block
             Predecessors (0)
             Statements (1)
-                IInvocationOperation (virtual void C.Enumerator.Dispose()) (OperationKind.Invocation, Type: System.Void, IsImplicit) (Syntax: 'new C()')
+                IInvocationOperation ( void C.Enumerator.Dispose()) (OperationKind.Invocation, Type: System.Void, IsImplicit) (Syntax: 'new C()')
```

Comment:
It doesn't look like any CFG coverage added for arguments with `foreach`

Reply:
Added `Foreach_RefStructEnumerator_DefaultDisposeArguments`

---
In reply to: [539538730](https://github.com/dotnet/roslyn/pull/49842#discussion_r539538730) [](ancestors = 539538730)